mitodl/mitxonline
# fix: limit user full name to 255 characters

#### Pre-Flight checklist


#### What are the relevant tickets?
#1438 

#### What's this PR do?
- Limits user's full name to have a max of 255 characters
- changes user name filed from Text to Char
- Adds 255 character limit to user profile form on the frontend
- adds a migration to trim existing full names longer than 255 characters

#### How should this be manually tested?
- Setup on master and create an account that has a full name containing more than 255 characters
- Shift to this branch, build and run the containers again, and make sure the new migration runs successfully and it trims the full name in the above point to 255 characters
- Register a new account, and check that you see a validation error message for the `Full Name` when you enter more than 255 characters

#### Where should the reviewer start?
- Setup on master, register an account

#### Screenshots
**Before**
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/34372316/219708180-b1490329-f16c-46d5-93f9-586b5d5d6bc9.png">

**After**
<img width="1787" alt="image" src="https://user-images.githubusercontent.com/34372316/220341907-84f11435-61be-4307-ab90-b9d5da422180.png">

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/34372316/220342149-afd9a891-21b9-4178-88fd-f224d2190105.png">
